### ISSUE TITLE
Server Error 500 when applying DO JSON in 1.9.0

### ISSUE THREAD

**Issue by ToniCipriani** [F5Networks/f5-declarative-onboarding#100]
### Environment
 * Declarative Onboarding Version: 1.9
 * BIG-IP Version: 15.0.1 Azure

### Summary
Attempting to build a cluster of Azure LB with 2 F5 BIG-IP Nodes in avail set via Terraform. When using DO, DO returns with a 500 server error

"errors":["Cannot read property 'includes' of undefined","Cannot read property 'includes' of undefined"]

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "schemaVersion": "1.9.0",
    "class": "Device",
    "label": "Basic onboarding",
    "Common": {
        "class": "Tenant",
        "hostname": "(redacted)",
        "dbvars": {
            "class": "DbVariables",
            "ui.advisory.enabled": true,
            "ui.advisory.color": "green",
            "ui.advisory.text": "/Common/hostname",
            "configsync.allowmanagement": "enable"
        },
        "myDns": {
            "class": "DNS",
            "nameServers": [
                "10.69.194.132",
                "10.69.194.133",
                "168.63.129.16"
            ],
            "search": [
                (redacted)
            ]
        },
        "myNtp": {
            "class": "NTP",
            "servers": [
                "time.nrc.ca",
                "0.pool.ntp.org",
                "1.pool.ntp.org"
            ],
            "timezone": "Canada/Eastern"
        },
        "myProvisioning": {
            "class": "Provision",
            "ltm": "nominal"
        },
        "default": {
            "class": "Route",
            "gw": "10.69.193.1",
            "network": "default",
            "mtu": 1500
        },
        "configsync": {
            "class": "ConfigSync",
            "configsyncIp": "10.69.193.15"
        },
        "failoverAddress": {
            "class": "FailoverUnicast",
            "address": "10.69.193.15"
        },
        "failoverGroup": {
            "class": "DeviceGroup",
            "type": "sync-failover",
            "members": ["(redacted, this is node1 fqdn)", "(redacted, this is node2 fqdn)"],
            "owner": "/Common/failoverGroup/members/0",
            "autoSync": true,
            "saveOnAutoSync": false,
            "networkFailover": true,
            "fullLoadOnSync": false,
            "asmSync": false
        },
        "trust": {
            "class": "DeviceTrust",
            "localUsername": "(redacted)",
            "localPassword": "(redacted)",
            "remoteHost": "10.69.193.11",
            "remoteUsername": "(redacted)",
            "remotePassword": "(redacted)"
        }
    }
}
```

2. Observe the following error response:
```json
"selfLink":"https://localhost/mgmt/shared/declarative-onboarding/task/d48587a5-3301-47b1-86c6-51f235f34ef8","code":500,"status":"ERROR","message":"invalid config - rolled back","errors":["Cannot read property 'includes' of undefined","Cannot read property 'includes' of undefined"],"result":{"class":"Result","code":500,"status":"ERROR","message":"invalid config - rolled back","errors":["Cannot read property 'includes' of undefined","Cannot read property 'includes' of undefined"]}
```

### Expected Behavior
200 OK 

### Actual Behavior
500 Internal Server Error

**Comment by dstokesf5** [comment]
I am having a hard time reproducing this failure. Can you try removing parts of the declaration to create a simpler declaration to try to reproduce this with? That simpler declaration could also help us narrow down the problem without needing to reproduce it.

**Comment by ToniCipriani** [comment]
I'll try re-running this with 1.9.0, but the same JSON file worked with 1.8.

**Comment by ToniCipriani** [comment]
Another issue I'm having with the same JSON is that after running this on a 1-NIC in Azure VE, the failoverGroup and Trust parts do not get applied, but I'll raise another issue.

**Comment by ToniCipriani** [comment]
@dstokesf5, did a bit more testing:

- I've narrowed down to the offending block to be the default route, which I removed, I cannot seem to define anything related to interface, VNET or routes in the JSON in either 1.8 or 1.9

- The initial apply on a fresh device fails now with a 400:
{"code":400,"message":"remoteSender:10.10.1.5, method:POST ","originalRequestBody":"{\"code\":400,\"message\":\"remoteSender:10.10.1.5, method:POST \",\"originalRequestBody\":\"{    \\\"schemaVersion\\\": \\\"1.8.0\\\",    \\\"class\\\": \\\"Device\\\",    \\\"label\\\": \\\"Basic onboarding\\\",    \\\"Common\\\": {        \\\"class\\\": \\\"Tenant\\\",        \\\"hostname\\\": \\\"(redacted)\\\",        \\\"dbvars\\\": {            \\\"class\\\": \\\"DbVariables\\\",            \\\"ui.advisory.enabled\\\": true,            \\\"ui.advisory.color\\\": \\\"green\\\",            \\\"ui.advisory.text\\\": \\\"/Common/hostname\\\",            \\\"configsync.allowmanagement\\\": \\\"enable\\\",            \\\"provision.1nicautoconfig\\\": \\\"disable\\\",            \\\"provision.1nic\\\": \\\"forced_enable\\\"        },        \\\"myDns\\\": {            \\\"class\\\": \\\"DNS\\\",            \\\"nameServers\\\": [                \\\"10.69.194.132\\\",                \\\"10.69.194.133\\\",                \\\"168.63.129.16\\\",                \\\"2606:4700:4700::1111\\\"            ],            \\\"search\\\": [                \\\"(redacted)\\\"            ]        },        \\\"myNtp\\\": {            \\\"class\\\": \\\"NTP\\\",            \\\"servers\\\": [                \\\"time.nrc.ca\\\",                \\\"0.pool.ntp.org\\\",                \\\"1.pool.ntp.org\\\"            ],            \\\"timezone\\\": \\\"Canada/Eastern\\\"        },        \\\"myProvisioning\\\": {            \\\"class\\\": \\\"Provision\\\",            \\\"ltm\\\": \\\"nominal\\\"        },        \\\"configsync\\\": {            \\\"class\\\": \\\"ConfigSync\\\",            \\\"configsyncIp\\\": \\\"10.69.193.251\\\"        },        \\\"failoverGroup\\\": {            \\\"class\\\": \\\"DeviceGroup\\\",            \\\"type\\\": \\\"sync-failover\\\",            \\\"members\\\": [\\\"(redacted)\\\"],            \\\"owner\\\": \\\"/Common/failoverGroup/members/0\\\",            \\\"autoSync\\\": true,            \\\"saveOnAutoSync\\\": false,            \\\"networkFailover\\\": false,            \\\"fullLoadOnSync\\\": false,            \\\"asmSync\\\": false        },        \\\"trust\\\": {            \\\"class\\\": \\\"DeviceTrust\\\",            \\\"localUsername\\\": \\\"(redacted)\\\",            \\\"localP","referer":"10.10.1.5","restOperationId":1749826,"kind":":resterrorresponse"}

- However if I take the same JSON and re-apply it again, it gives me 200 with no changes:

> * Connection #0 to host 10.69.193.250 left intact
> module.load_balancer_test.null_resource.vm00_run_DO_REST (local-exec): {"id":"58abfc20-bd27-41a9-9284-8d1f638cc93d","selfLink":"https://localhost/mgmt/shared/declarative-onboarding/task/58abfc20-bd27-41a9-9284-8d1f638cc93d","result":{"class":"Result","code":200,"status":"OK","message":"success"},"declaration":{"schemaVersion":"1.8.0","class":"Device","label":"Basic onboarding","Common":{"class":"Tenant","hostname":"(redacted)","dbvars":{"class":"DbVariables","ui.advisory.enabled":true,"ui.advisory.color":"green","ui.advisory.text":"/Common/hostname","configsync.allowmanagement":"enable","provision.1nicautoconfig":"disable","provision.1nic":"forced_enable"},"myDns":{"class":"DNS","nameServers":["10.69.194.132","10.69.194.133","168.63.129.16","2606:4700:4700::1111"],"search":["(redacted)"]},"myNtp":{"class":"NTP","servers":["time.nrc.ca","0.pool.ntp.org","1.pool.ntp.org"],"timezone":"Canada/Eastern"},"myProvisioning":{"class":"Provision","ltm":"nominal"},"configsync":{"class":"ConfigSync","configsyncIp":"10.69.193.250"},"failoverGroup":{"class":"DeviceGroup","type":"sync-failover","members":[(redacted)],"owner":"/Common/failoverGroup/members/0","autoSync":true,"saveOnAutoSync":false,"networkFailover":false,"fullLoadOnSync":false,"asmSync":false},"trust":{"class":"DeviceTrust","localUsername":"redacted","remoteHost":"/Common/failoverGroup/members/0","remoteUsername":"redacted"}},"async":false}}

- However if I do an inspect on the applied DO, it gives 412

[{"id":0,"selfLink":"https://localhost/mgmt/shared/declarative-onboarding/inspect","code":412,"status":"ERROR","message":"Precondition failed","errors":["Declaration contains INVALID items (suffixed with INVALID_X)","Unable to verify declaration from existing state.",{"keyword":"enum","dataPath":".declaration.Common['currentProvision']","schemaPath":"#/allOf/2/then/propertyNames/enum","params":{"allowedValues":["class","afm","am","apm","asm","avr","dos","fps","gtm","ilx","lc","ltm","pem","swg","urldb"]},"message":"should be equal to one of the allowed values","propertyName":"cgnat"},{"keyword":"propertyNames","dataPath":".declaration.Common['currentProvision']","schemaPath":"#/allOf/2/then/propertyNames","params":{"propertyName":"cgnat"},"message":"property name 'cgnat' is invalid"}],"result":{"class":"Result","code":412,"status":"ERROR","message":"Precondition failed","errors":["Declaration contains INVALID items (suffixed with INVALID_X)","Unable to verify declaration from existing state.",{"keyword":"enum","dataPath":".declaration.Common['currentProvision']","schemaPath":"#/allOf/2/then/propertyNames/enum","params":{"allowedValues":["class","afm","am","apm","asm","avr","dos","fps","gtm","ilx","lc","ltm","pem","swg","urldb"]},"message":"should be equal to one of the allowed values","propertyName":"cgnat"},{"keyword":"propertyNames","dataPath":".declaration.Common['currentProvision']","schemaPath":"#/allOf/2/then/propertyNames","params":{"propertyName":"cgnat"},"message":"property name 'cgnat' is invalid"}]},"declaration":{"class":"DO","declaration":{"class":"Device","schemaVersion":"1.9.0","Common":{"class":"Tenant","currentSystem":{"consoleInactivityTimeout":0,"hostname":"(redacted)","cliInactivityTimeout":0,"class":"System"},"currentProvision":{"afm":"none","am":"none","apm":"none","asm":"none","avr":"none","cgnat":"none","dos":"none","fps":"none","gtm":"none","ilx":"none","lc":"none","ltm":"nominal","pem":"none","swg":"none","urldb":"none","class":"Provision"},"currentNTP":{"servers":["time.nrc.ca","0.pool.ntp.org","1.pool.ntp.org"],"timezone":"Canada/Eastern","class":"NTP"},"currentDNS":{"nameServers":["10.69.194.132","10.69.194.133","168.63.129.16","2606:4700:4700::1111"],"search":["(redacted)"],"class":"DNS"},"internal":{"cmpHash":"default","mtu":1500,"tag":4094,"interfaces":[{"name":"1.0","tagged":false}],"class":"VLAN"},"self_1nic":{"address":"10.69.193.250/24","trafficGroup":"traffic-group-local-only","vlan":"internal","allowService":"default","class":"SelfIp"},"currentConfigSync":{"configsyncIp":"10.69.193.250","class":"ConfigSync"},"default_INVALID_1":{"gw":"10.69.193.1","mtu":0,"network":"default","class":"Route"},"default_INVALID_2":{"mtu":0,"network":"default","gw":"10.69.193.1","class":"ManagementRoute"},"dhclient_route1_INVALID_1":{"gw":"10.69.193.1","mtu":0,"network":"168.63.129.16/32","class":"Route"},"dhclient_route1_INVALID_2":{"mtu":0,"network":"168.63.129.16/32","gw":"10.69.193.1","class":"ManagementRoute"},"currentAuthentication":{"fallback":false,"enabledSourceType":"local","remoteUsersDefaults":{"role":"no-access","partitionAccess":"all","terminalAccess":"disabled"},"class":"Authentication"},"rd0":{"connectionLimit":0,"id":0,"strict":true,"vlans":["/Common/http-tunnel","/Common/socks-tunnel","/Common/internal"],"class":"RouteDomain"},"currentSnmpTrapEvents":{"device":true,"authentication":false,"agentStartStop":true,"class":"SnmpTrapEvents"},"currentSnmpAgent":{"contact":"Customer Name <admin@customer.com>","location":"Network Closet 1","allowList":["127.0.0.0/8"],"class":"SnmpAgent"},"comm-public":{"name":"public","access":"ro","ipv6":false,"source":"default","class":"SnmpCommunity"},"currentDagGlobals":{"icmpHash":"icmp","roundRobinMode":"global","ipv6PrefixLength":128,"class":"DagGlobals"},"currentHTTPD":{"allow":["all"],"authPamIdleTimeout":1200,"maxClients":10,"sslCiphersuite":["ECDHE-RSA-AES128-GCM-SHA256","ECDHE-RSA-AES256-GCM-SHA384","ECDHE-RSA-AES128-SHA","ECDHE-RSA-AES256-SHA","ECDHE-RSA-AES128-SHA256","ECDHE-RSA-AES256-SHA384","ECDHE-ECDSA-AES128-GCM-SHA256","ECDHE-ECDSA-AES256-GCM-SHA384","ECDHE-ECDSA-AES128-SHA","ECDHE-ECDSA-AES256-SHA","ECDHE-ECDSA-AES128-SHA256","ECDHE-ECDSA-AES256-SHA384","AES128-GCM-SHA256","AES256-GCM-SHA384","AES128-SHA","AES256-SHA","AES128-SHA256","AES256-SHA256"],"sslProtocol":"all -SSLv2 -SSLv3 -TLSv1","class":"HTTPD"},"currentTrafficControl":{"acceptIpOptions":false,"acceptIpSourceRoute":false,"allowIpSourceRoute":false,"continueMatching":false,"maxIcmpRate":100,"maxRejectRate":250,"maxRejectRateTimeout":30,"minPathMtu":296,"pathMtuDiscovery":true,"portFindThresholdTimeout":30,"portFindThresholdTrigger":8,"portFindThresholdWarning":true,"rejectUnmatched":true,"maxPortFindLinear":16,"maxPortFindRandom":16,"class":"TrafficControl"},"currentSSHD":{"inactivityTimeout":0,"class":"SSHD"}}}}}]

-Even with the 200, the config only gets applied up to the NTP. DNS, configSync, trust and failoverGroup settings are not applied. However DNS does get applied if I re-apply the exact same configuration a third time.

I suspect this is all due to the fact that I am building 1-NIC nodes (i.e. traffic/management all on the same IP), where DO doesn't like?

**Comment by bloodsun** [comment]
Sorry to jump into this thread.
I am able to reproduce the same error message on v1.10.0 with the following steps:

1. On a blank VE, add a default management route before Declarative Onboarding RPM  is installed.
```
[root@localhost:NO LICENSE:Standalone] config # tmsh list sys management-route                                                                                                                                                                                             
sys management-route default-inet6 {
    gateway 2001:db8:1::ffff
    network default-inet6
}
```

2. Install Declarative Onboarding RPM.

3. Perform the following declaration
```
curl -k -u 'admin:' -X POST -H "Content-type: application/json" https://localhost/mgmt/shared/declarative-onboarding -d '
{
    "schemaVersion": "1.9.0",
    "class": "Device",
    "async": true,
    "label": "BIG-IP declaration for declarative onboarding",
    "Common": {
        "class": "Tenant",
        "mgmt_route_1": {
            "class": "ManagementRoute",
            "mtu": 0,
            "gw": "2001:db8:1::ffff",
            "network": "2001:db8:0101:5414:1e01:0002:0000:0001/128"
        },
            "mgmt_route_2": {
            "class": "ManagementRoute",
            "mtu": 0,
            "gw": "2001:db8:1::ffff",
            "network": "2001:db8:0101:5414:1e01:0002:0000:0002/128"
        }
    }
}' | jq .
```
This is the result :
```
[root@localhost:NO LICENSE:Standalone] config # curl -k -u 'admin:' https://localhost/mgmt/shared/declarative-onboarding
{
  "id": "4ad0d3ec-d8af-435b-b7f9-6c5b2dd8131d",
  "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/4ad0d3ec-d8af-435b-b7f9-6c5b2dd8131d",
  "code": 500,
  "status": "ERROR",
  "message": "invalid config - rolled back",
  "errors": [
    "Cannot read property 'includes' of undefined",
    "Cannot read property 'includes' of undefined"
  ],
  "result": {
    "class": "Result",
    "code": 500,
    "status": "ERROR",
    "message": "invalid config - rolled back",
    "errors": [
      "Cannot read property 'includes' of undefined",
      "Cannot read property 'includes' of undefined"
    ]
  },
  "declaration": {
    "schemaVersion": "1.9.0",
    "class": "Device",
    "async": true,
    "label": "BIG-IP declaration for declarative onboarding",
    "Common": {
      "class": "Tenant",
      "mgmt_route_1": {
        "class": "ManagementRoute",
        "mtu": 0,
        "gw": "2001:db8:1::ffff",
        "network": "2001:db8:0101:5414:1e01:0002:0000:0001/128"
      },
      "mgmt_route_2": {
        "class": "ManagementRoute",
        "mtu": 0,
        "gw": "2001:db8:1::ffff",
        "network": "2001:db8:0101:5414:1e01:0002:0000:0002/128"
      }
    }
  }
}
```
Following is the full trace:
```
[root@localhost:NO LICENSE:Standalone] config # curl -k -u 'admin:' https://localhost/mgmt/shared/declarative-onboarding?show=full | jq . 
{
  "id": "4ad0d3ec-d8af-435b-b7f9-6c5b2dd8131d",
  "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/4ad0d3ec-d8af-435b-b7f9-6c5b2dd8131d",
  "code": 500,
  "status": "ERROR",
  "message": "invalid config - rolled back",
  "errors": [
    "Cannot read property 'includes' of undefined",
    "Cannot read property 'includes' of undefined"
  ],
  "result": {
    "class": "Result",
    "code": 500,
    "status": "ERROR",
    "message": "invalid config - rolled back",
    "errors": [
      "Cannot read property 'includes' of undefined",
      "Cannot read property 'includes' of undefined"
    ]
  },
  "declaration": {
    "schemaVersion": "1.9.0",
    "class": "Device",
    "async": true,
    "label": "BIG-IP declaration for declarative onboarding",
    "Common": {
      "class": "Tenant",
      "mgmt_route_1": {
        "class": "ManagementRoute",
        "mtu": 0,
        "gw": "2001:db8:1::ffff",
        "network": "2001:db8:0101:5414:1e01:0002:0000:0001/128"
      },
      "mgmt_route_2": {
        "class": "ManagementRoute",
        "mtu": 0,
        "gw": "2001:db8:1::ffff",
        "network": "2001:db8:0101:5414:1e01:0002:0000:0002/128"
      }
    }
  },
  "currentConfig": {
    "parsed": true,
    "Common": {
      "System": {
        "consoleInactivityTimeout": 0,
        "hostname": "bigip1",
        "cliInactivityTimeout": 0,
        "autoPhonehome": true
      },
      "Provision": {
        "afm": "none",
        "am": "none",
        "apm": "none",
        "asm": "none",
        "avr": "none",
        "dos": "none",
        "fps": "none",
        "gtm": "none",
        "ilx": "none",
        "lc": "none",
        "ltm": "nominal",
        "pem": "none",
        "sslo": "none",
        "swg": "none",
        "urldb": "none"
      },
      "NTP": {
        "timezone": "America/Los_Angeles"
      },
      "DNS": {
        "search": [
          "localhost"
        ]
      },
      "Trunk": {},
      "VLAN": {},
      "SelfIp": {},
      "Route": {},
      "ConfigSync": {
        "configsyncIp": "none"
      },
      "FailoverUnicast": {
        "unicastAddress": "none"
      },
      "ManagementRoute": {
        "default-inet6": {
          "name": "default-inet6",
          "mtu": 0,
          "network": "default-inet6",
          "gw": "2001:db8:1::ffff"
        }
      },
      "SyslogRemoteServer": {},
      "Authentication": {
        "fallback": false,
        "enabledSourceType": "local",
        "remoteUsersDefaults": {
          "role": "no-access",
          "partitionAccess": "all",
          "terminalAccess": "disabled"
        }
      },
      "RouteDomain": {
        "0": {
          "name": "0",
          "connectionLimit": 0,
          "id": 0,
          "strict": true,
          "vlans": [
            "/Common/http-tunnel",
            "/Common/socks-tunnel"
          ]
        }
      },
      "RemoteAuthRole": {},
      "SnmpTrapEvents": {
        "device": true,
        "authentication": false,
        "agentStartStop": true
      },
      "SnmpTrapDestination": {},
      "SnmpAgent": {
        "contact": "Customer Name <admin@customer.com>",
        "location": "Network Closet 1",
        "allowList": [
          "127.0.0.0/8"
        ]
      },
      "SnmpUser": {},
      "SnmpCommunity": {
        "comm-public": {
          "name": "public",
          "access": "ro",
          "ipv6": false,
          "source": "default"
        }
      },
      "DagGlobals": {
        "icmpHash": "icmp",
        "roundRobinMode": "global",
        "ipv6PrefixLength": 128
      },
      "HTTPD": {
        "allow": [
          "All"
        ],
        "authPamIdleTimeout": 1200,
        "maxClients": 10,
        "sslCiphersuite": [
          "ECDHE-RSA-AES128-GCM-SHA256",
          "ECDHE-RSA-AES256-GCM-SHA384",
          "ECDHE-RSA-AES128-SHA",
          "ECDHE-RSA-AES256-SHA",
          "ECDHE-RSA-AES128-SHA256",
          "ECDHE-RSA-AES256-SHA384",
          "ECDHE-ECDSA-AES128-GCM-SHA256",
          "ECDHE-ECDSA-AES256-GCM-SHA384",
          "ECDHE-ECDSA-AES128-SHA",
          "ECDHE-ECDSA-AES256-SHA",
          "ECDHE-ECDSA-AES128-SHA256",
          "ECDHE-ECDSA-AES256-SHA384",
          "AES128-GCM-SHA256",
          "AES256-GCM-SHA384",
          "AES128-SHA",
          "AES256-SHA",
          "AES128-SHA256",
          "AES256-SHA256"
        ],
        "sslProtocol": "all -SSLv2 -SSLv3 -TLSv1"
      },
      "TrafficControl": {
        "acceptIpOptions": false,
        "acceptIpSourceRoute": false,
        "allowIpSourceRoute": false,
        "continueMatching": false,
        "maxIcmpRate": 100,
        "maxRejectRate": 250,
        "maxRejectRateTimeout": 30,
        "minPathMtu": 296,
        "pathMtuDiscovery": true,
        "portFindThresholdTimeout": 30,
        "portFindThresholdTrigger": 8,
        "portFindThresholdWarning": true,
        "rejectUnmatched": true,
        "maxPortFindLinear": 16,
        "maxPortFindRandom": 16
      },
      "SSHD": {
        "inactivityTimeout": 0
      }
    }
  },
  "originalConfig": {
    "parsed": true,
    "Common": {
      "System": {
        "consoleInactivityTimeout": 0,
        "hostname": "bigip1",
        "cliInactivityTimeout": 0,
        "autoPhonehome": true
      },
      "Provision": {
        "afm": "none",
        "am": "none",
        "apm": "none",
        "asm": "none",
        "avr": "none",
        "dos": "none",
        "fps": "none",
        "gtm": "none",
        "ilx": "none",
        "lc": "none",
        "ltm": "nominal",
        "pem": "none",
        "sslo": "none",
        "swg": "none",
        "urldb": "none"
      },
      "NTP": {
        "timezone": "America/Los_Angeles"
      },
      "DNS": {
        "search": [
          "localhost"
        ]
      },
      "Trunk": {},
      "VLAN": {},
      "SelfIp": {},
      "Route": {},
      "ConfigSync": {
        "configsyncIp": "none"
      },
      "FailoverUnicast": {
        "unicastAddress": "none"
      },
      "ManagementRoute": {
        "default-inet6": {
          "name": "default-inet6",
          "mtu": 0,
          "network": "default-inet6",
          "gw": "2001:db8:1::ffff"
        }
      },
      "SyslogRemoteServer": {},
      "Authentication": {
        "fallback": false,
        "enabledSourceType": "local",
        "remoteUsersDefaults": {
          "role": "no-access",
          "partitionAccess": "all",
          "terminalAccess": "disabled"
        }
      },
      "RouteDomain": {
        "0": {
          "name": "0",
          "connectionLimit": 0,
          "id": 0,
          "strict": true,
          "vlans": [
            "/Common/http-tunnel",
            "/Common/socks-tunnel"
          ]
        }
      },
      "RemoteAuthRole": {},
      "SnmpTrapEvents": {
        "device": true,
        "authentication": false,
        "agentStartStop": true
      },
      "SnmpTrapDestination": {},
      "SnmpAgent": {
        "contact": "Customer Name <admin@customer.com>",
        "location": "Network Closet 1",
        "allowList": [
          "127.0.0.0/8"
        ]
      },
      "SnmpUser": {},
      "SnmpCommunity": {
        "comm-public": {
          "name": "public",
          "access": "ro",
          "ipv6": false,
          "source": "default"
        }
      },
      "DagGlobals": {
        "icmpHash": "icmp",
        "roundRobinMode": "global",
        "ipv6PrefixLength": 128
      },
      "HTTPD": {
        "allow": [
          "All"
        ],
        "authPamIdleTimeout": 1200,
        "maxClients": 10,
        "sslCiphersuite": [
          "ECDHE-RSA-AES128-GCM-SHA256",
          "ECDHE-RSA-AES256-GCM-SHA384",
          "ECDHE-RSA-AES128-SHA",
          "ECDHE-RSA-AES256-SHA",
          "ECDHE-RSA-AES128-SHA256",
          "ECDHE-RSA-AES256-SHA384",
          "ECDHE-ECDSA-AES128-GCM-SHA256",
          "ECDHE-ECDSA-AES256-GCM-SHA384",
          "ECDHE-ECDSA-AES128-SHA",
          "ECDHE-ECDSA-AES256-SHA",
          "ECDHE-ECDSA-AES128-SHA256",
          "ECDHE-ECDSA-AES256-SHA384",
          "AES128-GCM-SHA256",
          "AES256-GCM-SHA384",
          "AES128-SHA",
          "AES256-SHA",
          "AES128-SHA256",
          "AES256-SHA256"
        ],
        "sslProtocol": "all -SSLv2 -SSLv3 -TLSv1"
      },
      "TrafficControl": {
        "acceptIpOptions": false,
        "acceptIpSourceRoute": false,
        "allowIpSourceRoute": false,
        "continueMatching": false,
        "maxIcmpRate": 100,
        "maxRejectRate": 250,
        "maxRejectRateTimeout": 30,
        "minPathMtu": 296,
        "pathMtuDiscovery": true,
        "portFindThresholdTimeout": 30,
        "portFindThresholdTrigger": 8,
        "portFindThresholdWarning": true,
        "rejectUnmatched": true,
        "maxPortFindLinear": 16,
        "maxPortFindRandom": 16
      },
      "SSHD": {
        "inactivityTimeout": 0
      }
    }
  },
  "lastUpdate": "2020-01-29T15:48:27.137Z"
}
```

Question:
Is the management route not in declaration suppose to be removed/delete by DO similar to how AS3 treat the missing object in declaration?

**Comment by dstokesf5** [comment]
Issue is tracked internally as AUTOTOOL-1139.

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 1.11.0 release.
